ouj/python/algorithm/merge_sort.py

```
@@ -0,0 +1,59 @@
+                    r.append(ushiro[u])  # r に、ushiro の先頭を追加して
+                    u += 1  # u を一つ増やす
+    # print("out: ", r)
+    return r
+
+
+# 0以上999以下の乱数を kosu 個生成してリストにする
+kosu = 12
+min_r, max_r = 0, 999
+
+# min_r 以上 max_r以下の乱数を kosu個 生成する
+narabi = [rd.randint(min_r, max_r) for _ in range(kosu)]
```

Ruff (0.16.2)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)